src/frontend/components/PublicPausedLiveVideo/index.spec.tsx

```diff
@@ -3,10 +3,29 @@ import { DateTime } from 'luxon';
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 
+import { resumeLive } from '../../utils/resumeLive';
 import { videoMockFactory } from '../../utils/tests/factories';
 import { wrapInIntlProvider } from '../../utils/tests/intl';
 import { PublicPausedLiveVideo } from '.';
 
+jest.mock('video.js', () => ({
+  __esModule: true,
+  default: {
+    getPlayers: () => [
+      {
+        currentSource: () => 'https://live.m3u8',
+        src: jest.fn(),
+      },
+    ],
```

Comment:
In order to stick to reality:
```suggestion
    getPlayers: () => ({
      0: {
        currentSource: () => 'https://live.m3u8',
        src: jest.fn(),
      },
    }),
```

Reply:
True ! getPlayers returns an object keyed with the player ID.